pkgs/tools/networking/containerlab/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "containerlab";
+  version = "0.41.2";
+
+  src = fetchFromGitHub {
+    owner = "srl-labs";
+    repo = "containerlab";
+    rev = "v${version}";
+    hash = "sha256-HOsfQUCYlU19C30XKCQZj2Mw5NQfmp9miZYmN1s14QU=";
+  };
+
+  vendorHash = "sha256-dumXV5o/x4rqvX3tATsWFXz6DSowTSgaiHvzFoQaDIw=";
+
+  ldflags = [ "-s" "-w" "-X" "github.com/srl-labs/containerlab/cmd.version=${version}" "-X" "github.com/srl-labs/containerlab/cmd.commit=${src.rev}" "-X" "github.com/srl-labs/containerlab/cmd.date=1970-01-01T00:00:00Z" ];
+
+  meta = with lib; {
+    description = "container-based networking lab";
+    homepage = "https://containerlab.dev/";
+    changelog = "https://github.com/srl-labs/containerlab/releases/tag/v0.41.2";
```

```suggestion
    changelog = "https://github.com/srl-labs/containerlab/releases/tag/v${version}";
```

---

That's within the `with lib;` scope.
It does work, but from what I can tell, the following is slightly more preferred (lately):

```suggestion
    changelog = "https://github.com/srl-labs/containerlab/releases/tag/${src.rev}";
```

Furthermore, you could also drop the `with lib;` and use

```nix
{
    license = lib.licenses.bsd3;
    maintainers = with lib.maintainers; [ janik ];
}
```

:)

---

Is it fine now?